src/views/project/publishPartition/SummaryView.vue

```diff
@@ -0,0 +1,162 @@
+
+  const onetimeAction = formObject.onetimeAction || 1
+
+  const url = `${
+    import.meta.env.VITE_API_URL
+  }/projects/${projectId}/publish/partition/${partitionId}`
+  const response = await axios.post(url, { onetimeAction })
+
+  if (response.status == 200) {
+    alert('Publishing Partition')
+    router.push({ path: `/myprojects/${projectId}/overview` })
+  } else {
+    alert('Could not publish Partition')
+  }
+}
+</script>
+<template>
+  <LoadingIndicator :isLoaded="isLoaded">
+    <RouterLink :to="`/myprojects/${projectId}/publish/partition`">
+      <button type="button">Go Back</button>
+    </RouterLink>
+
+    <h1>Partition: {{ partition.name }}</h1>
+    <p>
+      This partition contains {{ characters }} characters, {{ taxa }} taxa,
+      {{ media }} media of which {{ onetimeMedia.length }} are released for
+      onetime use, {{ labels }} media labels, {{ specimens }} specimens,
+      {{ views }} views, {{ documents }} documents that are linked to media
+      explaining copyright and {{ bibliographicReferences }} bibliographic
+      references To edit these, go to the Matrix Editor and make changes
+    </p>
+    <p>
+      When you publish this partition, a new project will be created with the
+      characters and taxa listed above. Any matrices, media, copyright documents
+      and bibliographic citations associated with the characters and taxa will
+      be copied into the new project. The new project will be left in an
+      unpublished state to allow you time to review prior to publication.
+    </p>
+    <p>
+      Note that the new project you create here will be separate from P4091
+      (Test) and create only the subset of information described above. Changes
+      made to the newly created project will not affect P4091 or vice versa.
+    </p>
+    <p>
+      Note that publishing a partition may consume significant server resources,
+      particularly storage. Please be considerate and only publish when you need
+      to.
+    </p>
+
+    <div v-if="!taxa || !characters">
+      <p>
+        You cannot publish this partition because it is empty. Please add
+        characters and taxa and try again.
+      </p>
+    </div>
+
+    <div v-else>
+      <form @submit.prevent="publishPartition">
+        <div v-if="onetimeMedia.length">
+          <p>
+            This partition contains {onetimeMedia.length} media released for
+            onetime use. Due to this license, these media may not be usable in a
+            derivative project. You must either change the media reuse license,
+            or choose to keep the media in the current project, the new parition
+            based project or both (bypassing one-time use).
+          </p>
+          <li v-for="media in onetimeMedia" class="list-group-item">
+            <RouterLink
+              :to="`/myprojects/${projectId}/media/${media.media_id}`"
+            >
+              <button type="button">Onetime Media File</button>
```

Comment:
Rather than a button, let's make keep this as a text. The text should be "M{{ media.media_id }}